modules/fedimint-mint-client/src/lib.rs

```diff
@@ -1279,6 +1275,16 @@ impl MintClientModule {
     }
 }
 
+pub fn oob_spend_operation_id(
```

Comment:
Couldn't you even make this a member fn of [`OOBNotes`](https://docs.fedimint.org/fedimint_mint_client/struct.OOBNotes.html)? That's defined in the same crate afaik.

Reply:
tried this, `OOBNotes` has `FederationId` this function only. also changed the function name now.